ra2ce/analysis/losses/risk_calculation/risk_calculation_default.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC
+import geopandas as gpd
+import logging
+
+from ra2ce.analysis.losses.risk_calculation.risk_calculation_base import (
+    RiskCalculationBase,
+)
+
+
+class RiskCalculationDefault(RiskCalculationBase, ABC):
+    def __init__(self, risk_calculation_year: int, losses_gdf: gpd.GeoDataFrame):
+        super().__init__(risk_calculation_year, losses_gdf)
+        self._to_integrate = self._rework_damage_data()
```

Comment:
This is not needed, the calling to `self._rework_damage_data()` should be in the parent class.